### PR TITLE
Restructuring PeopleOps page

### PR DESCRIPTION
**Pull request by InesRoitman**

### CODE REVIEW

**Comment by InesRoitman** [comment]
Hey team! I made some changes to our handbook page (you can see 4 commits). Please let me know what you think :)

**Review by DebbieSG — APPROVED** [review]
Well done Ines!

**Review by CecilyBee — APPROVED** [review]
These look amazing, thank you Inés!

**Review by cassiemelani — APPROVED** [review]
LGTM :)

**Review by mercadon — APPROVED** [review]
This looks awesome, thanks for giving this section some much needed TLC :) 

My only super small suggestion would be to have the sections headings in "Things you need to know about working here" all follow the same format. Maybe we have them all be questions, like "are you switching teams?" or all follow the "How" format like "how we manage remote work." This is definitely not a blocker at all, I think the reformatting was a huge improvement!

**Review by dadlerj — APPROVED** [review]
Looks great! Just left a small suggestion for links for the future.

**Review by InesRoitman — COMMENTED** [review]
> Looks great! Just left a small suggestion for links for the future.

Yes! Thank you, Jean explained to me yesterday, so I'll work on that today (I'm excited)

File: handbook/index.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ The handbook is a living document and we expect every teammate to propose improv
 
 ## Teams at Sourcegraph
 
-### People Ops
+### People Ops[https://about.sourcegraph.com/handbook/people-ops]
```
[the remark this answers is not in the document]
Comment (InesRoitman) [reply handbook/index.md:26]: Yes! Thank you, Jean explained that to me yesterday, I'll work on that today (I'm excited)

We even made a video, so I'll add it to the handbook

**Pull request merged**